### ISSUE TITLE
upgrade

### ISSUE THREAD

**Issue by spea1** [Alignak-monitoring-contrib/alignak-webui#257]
```
root@nuc:~/repos/alignak-webui# git log

commit 9633ea87829be048baf2df3094b519f9cdc5974f
Merge: 3717311 2aa987e
Author: Frédéric MOHIER <frederic.mohier@gmail.com>
Date:   Sun Mar 26 10:57:48 2017 +0200

    Merge pull request #254 from Alignak-monitoring-contrib/update-edition

    Update edition mode
```

```	
root@nuc:~/repos/alignak-webui# sudo pip install -v --upgrade .

 Replacing /usr/local/lib/python2.7/dist-packages/test/test_web_external_embed.py
  Replacing /usr/local/lib/python2.7/dist-packages/test/test_web_external_embed.pyc
Cleaning up...
Invalid script entry point: <ExportEntry alignak-webui = alignak_webui.app:None []> for req: alignak-webui==0.7.6 - A callable suffix is required. Cf https://packaging.python.org/en/latest/distributing.html#console-scripts for more information.
Exception information:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/pip/basecommand.py", line 215, in main
    status = self.run(options, args)
  File "/usr/local/lib/python2.7/dist-packages/pip/commands/install.py", line 342, in run
    prefix=options.prefix_path,
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_set.py", line 784, in install
    **kwargs
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_install.py", line 851, in install
    self.move_wheel_files(self.source_dir, root=root, prefix=prefix)
  File "/usr/local/lib/python2.7/dist-packages/pip/req/req_install.py", line 1064, in move_wheel_files
    isolated=self.isolated,
  File "/usr/local/lib/python2.7/dist-packages/pip/wheel.py", line 492, in move_wheel_files
    maker.make_multiple(['%s = %s' % kv for kv in console.items()])
  File "/usr/local/lib/python2.7/dist-packages/pip/_vendor/distlib/scripts.py", line 383, in make_multiple
    filenames.extend(self.make(specification, options))
  File "/usr/local/lib/python2.7/dist-packages/pip/_vendor/distlib/scripts.py", line 372, in make
    self._make_script(entry, filenames, options=options)
  File "/usr/local/lib/python2.7/dist-packages/pip/_vendor/distlib/scripts.py", line 263, in _make_script
    script = self._get_script_text(entry).encode('utf-8')
  File "/usr/local/lib/python2.7/dist-packages/pip/wheel.py", line 405, in _get_script_text
    "information." % (entry, req)
InstallationError: Invalid script entry point: <ExportEntry alignak-webui = alignak_webui.app:None []> for req: alignak-webui==0.7.6 - A callable suffix is required. Cf https://packaging.python.org/en/latest/distributing.html#console-scripts for more information.
```

**Comment by mohierf** [comment]
The PR #255 will fix this problem. Will be merged very soon 😉

**Comment by mohierf** [comment]
@spea1 I merged #255 that should fix this problem

**Comment by mohierf** [comment]
I re-open this ...
